code/modules/clothing/glasses/glasses.dm

```diff
@@ -177,6 +177,15 @@
 	flags = NODROP
 	flags_cover = null
 
+/obj/item/clothing/glasses/material/lighting
+	name = "Neutron Goggles"

```

Comment:
wont' this mean they they can see objects through walls?

Reply:
Negative, as the sight_flags are reset on the subtype.